tests/kernel/mem_protect/mem_protect/src/mem_domain.c

```diff
@@ -130,8 +147,88 @@ void test_mem_domain_invalid_access(void *p1, void *p2, void *p3)
 
 	k_sem_take(&sync_sem, SYNC_SEM_TIMEOUT);
 }
+/***************************************************************************/
+static void thread_entry_rw(void *p1, void *p2, void *p3)
+{
+	u8_t read_data = mem_domain_buf[0];
+
+	/* Write to the partition */
```

Comment:
need to set valid_fault = true;

Reply:
Ok, I think valid_fault = false; I will modify. This is the case when the partition has both read/write permission.